test/e2e/tests/multichain/aggregated-balances.spec.ts

```diff
@@ -34,7 +34,6 @@ describe('Multichain Aggregated Balances', function (this: Suite) {
           .withEnabledNetworks({
             eip155: {
               [CHAIN_IDS.MAINNET]: true,
-              [CHAIN_IDS.SEPOLIA]: true,
```

Comment:
Test mocks have changed to be more realistic - we now only show 1 network at a time (unless if "all popular networks" is selected)